doc/mgmt/Docker to Host communication.md

```diff
@@ -2,7 +2,7 @@
 Docker to Host Communications
 
 # High Level Design Document
-#### Rev 0.7
+#### Rev 0.8
```

Comment:
Original document mentioned "The client application module in a container". We need to extend to client application running on the host.